flink-connectors/flink-connector-kafka/src/main/java/org/apache/flink/connector/kafka/source/reader/deserializer/KafkaRecordDeserializationSchema.java

```diff
@@ -108,24 +107,24 @@ default void open(DeserializationSchema.InitializationContext context) throws Ex
      */
     static <V> KafkaRecordDeserializationSchema<V> valueOnly(
             Class<? extends Deserializer<V>> valueDeserializerClass) {
-        return new KafkaValueOnlyDeserializerWrapper<>(
-                valueDeserializerClass, Collections.emptyMap());
+        return valueOnly(valueDeserializerClass, Collections.emptyMap());
     }
 
     /**
      * Wraps a Kafka {@link Deserializer} to a {@link KafkaRecordDeserializationSchema}.
      *
      * @param valueDeserializerClass the deserializer class used to deserialize the value.
-     * @param config the configuration of the value deserializer, only valid when the deserializer
-     *     is an implementation of {@code Configurable}.
+     * @param config the configuration of the value deserializer. If the deserializer is an
```

Comment:
nits: the doc seems to miss the explanation of what is invoked when the deserializer is NOT an implementation of Configurable.

Would it be slightly better to use something like this:
```
The configuration of the value deserializer. If the deserializer is an 
implementation of {@code Configurable}, 
{@link org.apache.kafka.common.Configurable#configure(Map)} 
will be invoked with the given config. Otherwise, {@link Deserializer#configure(Map, boolean)} 
will be invoked with the given config.
```